lib/chef/resource/powershell_package.rb

```diff
@@ -37,6 +37,7 @@ class PowershellPackage < Chef::Resource::Package
       property :package_name, [String, Array], coerce: proc { |x| [x].flatten }
       property :version, [String, Array], coerce: proc { |x| [x].flatten }
       property :source, [String]
+      property :skip_publisher_check, [true, false], default: false
```

Comment:
If you add introduced: '14.2' we can make sure this displays correctly on the docs site when we start autogenerating resource docs. You should also add description: 'something something' so we know how to document this on the docs site

Reply:
Added